src/Command/InfectionCommand.php

```diff
@@ -259,9 +259,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
         return $statusCode;
     }
 
-    private function includeUserBootstrap(InfectionConfig $config): void
+    private function includeUserBootstrap(InfectionConfig $infectionConfig): void
     {
-        $bootstrap = $config->getBootstrap();
+        $bootstrap = $infectionConfig->getBootstrap();
```

Comment:
maybe instead of changing names here the bootstrap file should be included in its own scope:

```php
$bootstrap = $config->getBootstrap();

(function ($_file) {
    // Include the file in a separate scope to avoid variable shadowing
    require_once $_file;
})($bootstrap);
```

Reply:
Or you can `unset($config)` before the inclusion.

Reply:
I think a closure will be safer to use: you don't have any change to do if you add a variable or parameter; but yeah ultimately both works so let's pick one and be done with it :)